src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-rpm-template

```diff
@@ -34,8 +34,6 @@
 # $JAVA_OPTS used in $RUN_CMD wrapper
 export JAVA_OPTS
 
-prog="${{exec}}"
```

Comment:
You mean change the var name `app_name` in the future? As you say, that would be pretty disastrous for backwards compatibility :)